docker-compose.yml

```
@@ -4,7 +4,33 @@ services:
+      - POSTGRES_PORT=5432
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=1234
+      - POSTGRES_DATABASE=paystell
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - PORT=4000
+      - NODE_ENV=production
+      - AUTH0_SECRET=your-auth0-secret
+      - AUTH0_BASE_URL=http://localhost:4000
+      - AUTH0_ISSUER_BASE_URL=https://dev-wu6ywz3z0p1tpbkp.us.auth0.com
+      - AUTH0_CLIENT_ID=dO2W6eUGpJmQx4IWhDtytcdG8nhej9mDg3g9Y4TAL-pyIootyoR8Z_jnB0gjUpiK   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)